ui/js/controllers/bugfiler.js

```diff
@@ -35,6 +35,24 @@ treeherder.controller('BugFilerCtrl', [
             $scope.reftest = reftest;
         }
 
+        $scope.thisTree = thisTree;
+        var integrationRepos = ['mozilla-inbound', 'fx-team', 'autoland'];
+        var releaseRepos = ['mozilla-aurora', 'mozilla-beta', 'mozilla-release'];
+        $scope.hasPossibleFileName = function() {
+            return $scope.possibleFilename.length > 0;
+        };
+        $scope.isAuroraOrOlder = function() {
+            return releaseRepos.indexOf(thisTree) >= 0;
+        };
+        $scope.isBetaOrOlder = function() {
+            return releaseRepos.indexOf(thisTree, 1) >= 0;
+        };
+        $scope.isReleaseOrOlder = function() {
+            return releaseRepos.indexOf(thisTree, 2) >= 0;
```

Comment:
I wouldn't hardcode the indexes, it's pretty fragile - make it a dynamic comparison (and add comments to `values.js` that the order matters; perhaps even add order ints).